src/app.ts

```diff
@@ -286,7 +286,7 @@ export const initializeAppModules = () => {
         action: function (output) {
           microsoftTeams.registerBeforeUnloadHandler(function (readyToUnload) {
             (window as any).readyToUnload = readyToUnload;
-            output("BeforeUnload recieved");
+            alert('beforeUnload recieved');
```

Comment:
You need to also run `yarn build` and check-in the updated files that the build produces. There is no CI/CD build for this web app.